graphql/schema.graphql

```diff
@@ -18,6 +18,7 @@ type Query {
   categories(treeLevel: Int = 3): [Category]
```

Comment:
@lucasrusso95 You must add this feature at [CHANGELOG.md](https://github.com/vtex-apps/store-graphql/blob/master/CHANGELOG.md) in [Unreleased] session.

Reply:
@brunojdo Added.